seq2seq/third_party/models/modeling_t5.py

```diff
@@ -14,409 +14,63 @@
 # limitations under the License.
```

Comment:
This file has been modified from its original, so text will need to be inserted at line 3 as described in the doc I sent you.

Reply:
done, thanks